### PR TITLE
STIJ-70: Added atom url field.

### PR DESCRIPTION
**Pull request by Jeroen005**

### CODE REVIEW

File: components/21-atoms/inputs/input-text/input-text.twig [hunk 1]
```diff
@@ -1 +1 @@
-<input type="text" name="textfield" id="textfield" placeholder="{{ placeholder|default(placeholder) }}" class="{{ modifier }}" }"/>
+<input type="text" name="textfield" id="textfield" placeholder="{{ placeholder|default(placeholder) }}" class="{{ modifier }}" />
```
Comment (meirege) [inline components/21-atoms/inputs/input-text/input-text.twig]: We should probably get those ID's out ;)

File: components/21-atoms/inputs/input-url/input-url.twig [hunk 2]
```diff
@@ -0,0 +1 @@
+<input type="url" name="url-field" id="url-field" placeholder="{{ placeholder|default(placeholder) }}" class="{{ modifier }}" />
```
Comment (meirege) [inline components/21-atoms/inputs/input-url/input-url.twig]: Same here, get the ID out is probably better.

File: components/21-atoms/inputs/input-text/input-text.twig (hunk 1 above)
Comment (Jeroen005) [reply components/21-atoms/inputs/input-text/input-text.twig]: This issue is already logged in STIJ-50. I think we should fix it there.

File: components/21-atoms/inputs/input-url/input-url.twig (hunk 2 above)
Comment (Jeroen005) [reply components/21-atoms/inputs/input-url/input-url.twig]: This issue is already logged in STIJ-50. I think we should fix it there.

**Pull request merged**